pom.xml

```diff
@@ -729,8 +756,26 @@
             <groupId>com.fasterxml.jackson.jaxrs</groupId>
             <artifactId>jackson-jaxrs-json-provider</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>org.glassfish.hk2.external</groupId>
+            <artifactId>javax.inject</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>ackson-module-jaxb-annotations</artifactId>
+          </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.jaxrs</groupId>
```

Comment:
What does this dependency do/why did we add these?

Reply:
This dependency is brought from two sources, and we only need one:
```
Dependency convergence error for com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.5.4 paths to dependency are:
+-com.linkedin.pinot:pinot-server:0.016
  +-org.glassfish.jersey.media:jersey-media-json-jackson:2.24
    +-com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.5.4
and
+-com.linkedin.pinot:pinot-server:0.016
  +-io.swagger:swagger-jaxrs:1.5.10
    +-com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.4.5
```

Reply:
does swagger still work when you make this change?

Reply:
I've tested it. It works after the changes.